### PR TITLE
FOUR-8972

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
Register some events related to user activity events according to the [PRD](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3084320783/User+Activity+Logging).
Make some changes related to label and getChanges

## Solution
- Improve the tickets definition

## How to Test
Check the actions related to 
     - CategoryCreated.php
        - CategoryUpdated.php
        - CustomizeUiUpdated.php
        - EnvironmentVariablesCreated.php
        - EnvironmentVariablesDeleted.php
        - EnvironmentVariablesUpdated.php
        - FilesUpdated.php
        - GroupCreated.php
        - ProcessArchived.php
        - ProcessCreated.php
        - ProcessPublished.php
        - ProcessRestored.php
        - ScreenCreated.php
        - ScreenDeleted.php
        - ScriptCreated.php
        - ScriptDeleted.php
        - ScriptExecutorCreated.php
        - ScriptExecutorDeleted.php
        - ScriptExecutorUpdated.php
        - ScriptUpdated.php
        - SettingsUpdated.php
        - TemplateCreated.php
        - TemplateUpdated.php
        - TokenCreated.php
        - TokenDeleted.php

## Related Tickets & Packages
[FOUR-8972](https://processmaker.atlassian.net/browse/FOUR-8972)

## Code Review Checklist
- [x] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [x] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [x] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [x] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [x] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Enviroment
ci:deploy

[FOUR-8972]: https://processmaker.atlassian.net/browse/FOUR-8972?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ProcessMaker/Events/EnvironmentVariablesCreated.php [hunk 1]
```diff
@@ -35,10 +35,10 @@ public function getData(): array
     {
         return [
             'name' => [
-                'label' => $this->variable['name'],
+                'label' => $this->variable['name'] ?? '',
```
Comment (fagubla) [inline ProcessMaker/Events/EnvironmentVariablesCreated.php:38]: This label must not be empty, because the modal will not display the link

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-b1d9f0e151.eng.processmaker.net

**Pull request merged**